api/src/main/java/org/eclipse/microprofile/jwt/algorithm/SignatureAlgorithm.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
+ *
+ *  See the NOTICE file(s) distributed with this work for additional
+ *  information regarding copyright ownership.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.jwt.algorithm;
+
+/**
+ * JWT JSON Web Signature Algorithms which must be supported.
+ * 
+ * @see <a href="https://tools.ietf.org/html/rfc7518#section-3">https://tools.ietf.org/html/rfc7518#section-3</a>
+ */
+public enum SignatureAlgorithm {
```

Comment:
Only used in TCK, so should not be in API I guess

Reply:
@rdebusscher I'd prefer in our implementation to use a property shipped with MP JWT to initialize `mp.jwt.publickey.algorithm`. Other implementations may follow suit too for the same purpose or might want to use it as part of their implementations.

Reply:
@rdebusscher I can remove it altogether rather than move it to TCK. If you can not comfortable with it being shipped in the api artifact then I'll just drop it.

Reply:
@rdebusscher OK, probably simpler to drop it for now, I had it copied from the builder PR, but that PR is bigger and will not make it to 1.2. so I'll just drop this enum for now